code/modules/species/species_bodytype_helpers.dm

```diff
@@ -31,10 +31,17 @@
 	return get_deform ? icon_deformed : icon_base
 
 /decl/bodytype/proc/get_tail(var/mob/living/carbon/human/H)
-	return tail
+	var/obj/item/organ/external/tail/tail = H.organs_by_name[BP_TAIL]
```

Comment:
Using bodytype helper procs at all seems strange if tails are being moved entirely to an organ - you might as well just make the tail appearance a property of the tail organ specifically and use tail subtypes to customize it per-species.

Reply:
Yes, I need to move that to the organ